draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx

```diff
@@ -199,1106 +199,175 @@ export const Assertive = ({
   return StaticIllustration
 }
 
+function createSpotIllustration(url: string) {
+  return ({ enableAspectRatio, ...props }: SpotProps) => (
+    <Base
+      aspectRatio={enableAspectRatio ? "square" : undefined}
+      {...props}
+      name={url}
+    />
+  )
+}
```

Comment:
```suggestion
const createSpotIllustration = (url: string): React.VFC<SpotProps> =>
  ({ enableAspectRatio, ...restProps }) => (
    <Base
      aspectRatio={enableAspectRatio ? "square" : undefined}
      {...restProps}
      name={url}
    />
  )
```

I think that should work?